### ISSUE TITLE
Conflict with the ReColor add-on

### ISSUE THREAD

**Issue by dollajas** [AnKingMed/Browser-Resizer#10]
This add on prevents me from color changing aspects of the browse window using the recolor add on, specifically the sidebar - wondering if this is fixable

Running 2.1.49 on the newest mac OS with no other add ons when I tested this

Recolor: https://ankiweb.net/shared/info/688199788

**Comment by AnKingMed** [comment]
Mine is working fine. What is your browser resizer config?

**Comment by dollajas** [comment]
Default config - we discussed reporting this as an issue since this add on rewrites CSS and prevents recolor from changing sidebar colors in browse

[Here was that original thread](https://www.reddit.com/r/Anki/comments/t3dh6d/comment/hysksec/?utm_source=share&utm_medium=web2x&context=3)

**Comment by AnKingMed** [comment]
Yeah I remember that. I've got the default config and recolor and my sidebar is being colored just fine. Do you have the latest version from ankiweb? (not a patreon version or anything?)

**Comment by dollajas** [comment]
Oh interesting, yeah I have the updated versions of everything. I don't mind ignoring it for now if it's just a local issue

**Comment by AnKingMed** [comment]
Weird. Can you add a picture of what it looks like? I can't reproduce it

**Comment by dollajas** [comment]
it appears darker, more in line with the card background color than the
rest of the browse

https://imgur.com/a/tPdYFMR

**Comment by AnKingMed** [comment]
Try changing the Frame Background to something like bright red

**Comment by dollajas** [comment]
changed it to maraschino red and the sidebar went along with it

https://imgur.com/ef7foJZ

**Comment by AnKingMed** [comment]
So it's working then?

**Comment by dollajas** [comment]
No, when I disable this add on it looks like this https://imgur.com/6S2rbae

The resizer add on is essentially turning the sidebar into the "frame
window"

**Comment by AnKingMed** [comment]
ah that makes sense now.  I'll have to investigate further

Also just FYI you can just paste or drag and drop images into here
